examples/process_gain.py

```diff
@@ -66,6 +53,19 @@ def determine_power_from_fit(filename, guessamp, guessph):
         amplitudes from fits
     """
     A, omega, phi, t = symbols("A omega phi t", real=True)
+    # {{{ Even though node names for both files should match, determine the
+    #     node names and resulting frequency coordinates separate for both
+    #     files.
+    all_node_names = sorted(
+        psd.find_file(
+            re.escape(filename),
+            exp_type=data_dir,
+            return_list=True,
+        ),
+        key=lambda x: float(x.split("_")[1]),
+    )
+    frq_kHz = np.array([float(j.split("_")[1]) for j in all_node_names])
+    # }}}
```

Comment:
this is part of the function and adds robustness incase the files DON'T have the same nodenames

Reply:
yes, this is mostly what I'm saying in the comment above